release/_bom.sh

```diff
@@ -285,10 +337,36 @@ function generate_poms {
 
 	rm -f ./*.pom
 
+	for pom in "release.${LIFERAY_RELEASE_PRODUCT_NAME}.api" "release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom" "release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.compile.only" "release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.third.party" "release.${LIFERAY_RELEASE_PRODUCT_NAME}.distro"
+	do
+		lc_download "https://repository.liferay.com/nexus/service/local/repositories/liferay-public-releases/content/com/liferay/portal/${pom}/${base_version}/${pom}-${base_version}.pom" "${pom}-${base_version}.pom"
+
+		sed \
+			-e "s#<version>${base_version}</version>#<version>${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}</version>#" \
+			-e "s#<connection>scm:git:git@github.com:liferay/liferay-portal.git</connection>#<connection>scm:git:git@github.com:liferay/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}.git</connection>#" \
+			-e "s#<developerConnection>scm:git:git@github.com:liferay/liferay-portal.git</developerConnection>#<developerConnection>scm:git:git@github.com:liferay/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}.git</developerConnection>#" \
+			-e "s#<tag>.*</tag>#<tag>${_PRODUCT_VERSION}</tag>#" \
+			-e "s#<url>https://github.com/liferay/liferay-portal</url>#<url>https://github.com/liferay/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}</url>#" \
+			-e "w ${pom}-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.pom" \
+			"${pom}-${base_version}.pom" > /dev/null
+
+		rm -f "${pom}-${base_version}.pom"
+	done
+}
+
+function generate_poms_from_scratch {
+	mkdir -p "${_BUILD_DIR}/boms"
+
+	lc_cd "${_BUILD_DIR}/boms"
+
+	rm -f ./*.pom
+	rm -f ./*.jar
+
```

Comment:
@pgydev, can you replace rebase your code and fix possible conflicts?

Right now on `master`, there is only a method named `generate_poms` since the method `generate_poms_from_scratch` was deleted.

Reply:
@natocesarrego Yes. I did the same here but i did not rename it and deleted the `generate_poms` because i didn't know for sure that you want to get rid of it.